src/olympia/templates/copyright.html

```diff
@@ -12,7 +12,7 @@
       </a>
       {% block extra_footer_links %}
         &nbsp;|&nbsp;<a href="https://status.mozilla.org">{{ _('Site Status')}}</a>
-        &nbsp;|&nbsp;<a href="https://developer.mozilla.org/en-US/Add-ons/AMO/Policy/Contact">{{ _('Report a bug')}}</a>
+        &nbsp;|&nbsp;<a href="https://github.com/thundernest/addons-server/issues/new">{{ _('Report a bug')}}</a>
```

Comment:
GitHub Issues isn't a good replacement for that contact page. developer.thunderbird.net needs an equivalent of the Mozilla add-on-related contacts page made.

Reply:
The splash page kind of has an equivalent to what that MDN page had: https://developer.thunderbird.net/ - though TB barely has any add-on specific resources (except for maybe a mailing list and that discourse forum). I wonder how @ryanleesipes would want do direct such communication that would come in via "report a bug".

Reply:
Nevermind, I didn't read the whole line and notice that it was for "Report a Bug". Issues are correct for that.

It is odd that AMO linked "Report a Bug" to a page with discussion forums and stuff like that, I don't want actual bug reports about the site to go anywhere else. I don't want addon authors asking addon development questions on here, but that's not actually very likely given the label.

This link is fine.